portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue

```diff
@@ -264,10 +264,10 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
                 </div>
               </td>
               <td class="badge-title-col">
-                <div>{{ $t(`badge.title.${badge.title}`,badge.title) }}</div>
+                <div>{{ $t(`badge.title.${badge.title.replace(' ','')}`,badge.title) }}</div>
```

Comment:
Why is this needed ? could you do a String.trim() on the badge object itself.

Reply:
`String.trim()` remove  only space from beginning and end,but we need to remove spaces from the middle